tests/Unit/EmogrifierTest.php

```diff
@@ -1739,10 +1739,12 @@ public function emogrifyMergesCssWithMixedCaseAttribute()
 
         $result = $this->subject->emogrify();
 
-        static::assertContains(
-            '<p style="margin: 0; padding-top: 1px; padding-bottom: 3px; text-align: center;">',
-            $result
-        );
+        // Property order in style attribute now follows precedence order, though used to follow application order
+        // Either is valid in this case
+        static::assertThat($result, static::logicalOr(
```

Comment:
I think using only the new version output would be enough here.

Reply:
Ok, have changed this.